hibernate-core/src/main/java/org/hibernate/boot/MetadataBuilder.java

```diff
@@ -225,6 +225,23 @@
 	MetadataBuilder enableNewIdentifierGeneratorSupport(boolean enable);
 
 	/**
+	 * Should we enable support for the "new" (since 5.2) type conflict resolving.
+	 *
+	 * Its default is defined by the {@link org.hibernate.cfg.AvailableSettings#USE_NEW_TYPE_CONFLICT_RESOLVER}
+	 * setting if using property-based configuration.
+	 *
+	 *
+	 * @param enable {@code true} to enable; {@code false} to disable;don't call for
+	 * default.
+	 *
+	 * @return {@code this}, for method chaining
+	 * @since 5.2.9
```

Comment:
In a micro release we cannot change an API signature, we can only add default methods.